Tasks/VsTest/package.json

```diff
@@ -1,27 +1,27 @@
 {
-    "name": "vsts-tasks-vstest",
-    "version": "1.0.0",
-    "description": "VSTS VSTEST Task",
-    "main": "VSTest.js",
-    "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git+https://github.com/Microsoft/vsts-tasks.git"
-    },
-    "author": "Microsoft Corporation",
-    "license": "MIT",
-    "bugs": {
-        "url": "https://github.com/Microsoft/vsts-tasks/issues"
-    },
-    "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
-    "dependencies": {
-        "xml2js": "0.4.16",
-        "vsts-task-lib": "^0.8.3",
-        "performance-now": "0.2.0"
-    },
-    "optionalDependencies": {
-        "regedit": "2.2.6"
-    }
+  "name": "vsts-tasks-vstest",
+  "version": "2.0.0",
```

Comment:
I would take this version out.  It's not worth maintaining and matching task.json.  Only the task.json is important since this is never published to an npm repository.